packages/gatsby-theme-docs/src/layouts/internals/layout-header.js

```diff
@@ -180,8 +181,71 @@ const SearchContainer = styled.div`
   }
 `;
 
+const SiteContextTitle = styled.div`
+  margin-right: ${designSystem.dimensions.spacings.s};
+`;
+
+const SiteTitle = styled.div`
+  margin-right: ${designSystem.dimensions.spacings.s};
+  ${({ hasContext }) =>
+    hasContext &&
+    `
+    font-weight: 600;
```

Comment:
Please use a design token for font weight and not a hardwired value.  Please use the one for semibold 

https://github.com/commercetools/commercetools-docs-kit/blob/main/packages/ui-kit/src/design-system.ts#L310